test/rubocop/cop/ruby_lsp/use_language_server_aliases_test.rb

```diff
@@ -0,0 +1,121 @@
+# typed: strict
+# frozen_string_literal: true
+
+require "test_helper"
+
+class UseLanguageServerAliasesTest < Minitest::Test
+  def test_registers_offense_when_using_interface_within_ruby_lsp
+    assert_offense(<<~RUBY)
+      module RubyLsp
+        sig { override.returns(T.all(T::Array[LanguageServer::Protocol::Interface::FoldingRange], Object)) }
+                                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Use constant alias `Interface`.
```

Comment:
I noticed the error seems to only include from `::Protocol` forward. Would it be possible to start the from `LanguageServer`?

Reply:
@vinistock I don't think I understand your question. What I see is that the error starts from `LanguageServer`. Do you want it to be something else?

![Screenshot 2023-03-12 at 10 07 26 PM](https://user-images.githubusercontent.com/4749355/224549924-f5bb8e2a-c13f-4695-91d5-1fb3903b54b2.png)

Reply:
Oh, interesting. It appears distorted to me, in my GitHub view it shows the error starting from `::Protocol` all the way to `FoldingRange` - which I couldn't understand since the node is supposed to be the namespace. Just a visual thing then, the code uses the right location.